README.md

```diff
@@ -17,19 +17,19 @@ Table of Contents
     * [License](#license)
 
 ## Usage
-Add it as an npm dev-dependency
+Add it as an npm devDependency, as well as all the necessary [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/).
 
 ```bash
-$ npm i eslint-config-commercetools --save-dev
+$ npm install --save-dev eslint-config-commercetools eslint@^#.#.# eslint-config-airbnb@^#.#.# eslint-plugin-jsx-a11y@^#.#.# eslint-plugin-import@^#.#.# eslint-plugin-react@^#.#.#
```

Comment:
> We can add it to 'postinstall script' though but I think there is a flag to ignore post-install scripts.
Unfortunately we can't use `postinstall` script (I tried it), because it will run within this package and will install those dependencies within this package (which is the root of the problem).

> So that means anybody using this library, will have to add this instruction in their CONTRIBUTING.md right?
This script is just a help for whoever wants to add this config to its project to install those deps, but in the project itself, not this package.

> do we really need to keep supporting npm 2
If you use node@4, npm@2 is the version you get. So for now we have to support this.